src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/VedtakOgBeregningBarnetilsyn.tsx

```diff
@@ -17,40 +24,66 @@ const Container = styled.div`
     gap: 1.5rem;
 `;
 
+const Skillelinje = styled.div`
+    border-left: 1px solid white;
+`;
+
 const VedtakOgBeregningBarnetilsyn: FC = () => {
+    const { request } = useApp();
+    const { behandling } = useBehandling();
     const { vedtak } = useVedtak();
 
+    const [vilkårsoppsummering, settVilkårsoppsummering] =
+        useState<Ressurs<Vilkårsoppsummering>>(byggTomRessurs());
     const [typeVedtak, settTypeVedtak] = useState<TypeVedtak | undefined>();
 
+    const hentVilkåroppsummering = useCallback(() => {
+        request<Vilkårsoppsummering, null>(`/api/sak/vilkarsoppsummering/${behandling.id}`).then(
+            settVilkårsoppsummering
+        );
+    }, [behandling, request]);
```

Comment:
Tror det kanskje er tilstrekkelig å ha dep-arr på `behandling.id` samtidig som det ikke burde ha så mye å si på denne siden

```suggestion
    }, [behandling.id. request]);
```